ipn/serve.go

```diff
@@ -38,7 +38,7 @@ type ServeConfig struct {
 	// traffic is allowed, from trusted ingress peers.
 	AllowFunnel map[HostPort]bool `json:",omitempty"`
 
-	// Foreground is a map of an IPN Bus session id to a
+	// Foreground is a map of an IPN Bus session ID to a
 	// foreground serve config. Note that only TCP and Web
 	// are used inside the Foreground map.
```

Comment:
why not AllowFunnel too? Doesn't that simplify your whole `shouldDeleteFunnel` code?

Reply:
Per our chat, we decided to just nest AllowFunnel in the foreground config